src/_data/sites/ja.yml

```
@@ -0,0 +1,542 @@
+          可能であればESLintのロゴとワードマークの両方を使用することが望ましいです。
+      download_svg: SVG ダウンロード
+      download_png: PNG ダウンロード
+      change_background: "背景色の変更"
+    color_palette:
+      title: カラーパレット
+      description:
+        - >
+          ロゴや文言と同じくらい、ESLintのカラーパレットにはブランドを表す力があります。
+          デザインの見た目に影響を与えるだけでなく、感情を呼び起こし、
+          ESLintブランドの個性を反映することができるのです。
+        - >
+          ブランドの中心となるカラーはボタン、リンク、入力コンポーネントなど、すべてのインタラクティブな要素で使用されます。
+          ロゴに由来しています。既存の2色を使って、独自の一次色調を作り出しています。
+      brand_palette: ブランド　パレット
+      colors:
+        primary: プライマリ
+        white: 白色
+        neutral: ニュートラル
+  actions:
+    back_to_home: ホームページへ
+    browse_docs: ドキュメンテーションを読む
+
+#------------------------------------------------------------------------------
+# Homepage
+#------------------------------------------------------------------------------
+
+homepage:
+  title: JavaScriptコードの問題点の発見と修正
+  description: >
+    ESLintは、コードを静的に解析し、問題を迅速に発見します。
+    ほとんどのテキストエディタに組み込まれており、CIパイプラインの一部としてESLintを実行することができます。
+
+  animation:
+    enabled: true
+    text: Find and fix problems in your JavaScript code
+    highlight_word: problems
+    wrong_word: problmes
+
+  install:
+    title: ESLint インストールコマンド
+    copy: インストールコマンドをコピーします
+    copied: コピーしました
+
+  versions:
+    title: ESLintバージョン
+    latest: 最新バージョン
+    upcoming: 次のバージョン
+    development: 開発中バージョン
+    dateline: <span class="text-dark">VERSION</span>（DATE）
+
+  explainer:
+    image:
+      src: eslint-features-img.png
+      alt: "ESLintが動作しているVisual Studioのコードウィンドウのスクリーンショット：IDE内のESLintエラーに下線を引いています。"
+    title: The pluggable linting utility for JavaScript and JSX
+    description: >
+      ESLintは、あなたのJavaScriptコードの問題を発見し、修正することを支援するオープンソースプロジェクトです。
+      ブラウザのためか、サーバのためか、フレームワークを使用しているかに関わらず、ESLintはあなたのコードが
+      良い状態になるように支援します。
+    features:
+      - title: 間違いを探す
+        description: >
+          ESLintは、コードを静的に解析し、問題を迅速に発見します。ESLintは ほとんどのテキストエディタに組み込まれており、
+          ESLintをCIパイプライン内で実行することもできます。
+        learn_more:
+          link: getStarted
+          text: >
+            <span class="visually-hidden">ESLintで間違いを探す方法について</span>もっと知りたい
+      - title: 自動的に修正する
+        description: >
+          ESLintが発見した問題のほとんどは、自動的に修正することができます。
+          ESLintは構文を意識して修正を行うため、従来の置換ベースのコード修正アルゴリズムのように新しいエラーを生み出すことはありません。
+        learn_more:
+          link: fixProblems
+          text: >
+            <span class="visually-hidden">ESLintの自動修正について</span>もっと知りたい
+      - title: Configure everything
+        description: >
+          コードの前処理、カスタムパーサーの使用、ESLintの組み込みルールと一緒に動作する独自のルールを作成することができます。
+          プロジェクトで必要な形に合わせてESLintをカスタマイズすることができます。
+        learn_more:
+          link: configuring
+          text: >
+            <span class="visually-hidden">ESLintのカスタマイズについて</span>もっと知りたい
+  blog:
+    view_all_posts: 全てのブログポストを見る
+
+  stats:
+    title: ようこそ、コミュニティへ
+    description: >
+      ESLintは、npmでのダウンロード数でNo.1のJavaScriptリンターです（DOWNLOAD_COUNT以上）。
```

I think the following expression is better:

```suggestion
      ESLintはnpmにおけるダウンロード数がNo.1のJavaScriptリンターです（DOWNLOAD_COUNT以上）。
```